docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -55,7 +55,9 @@ Pub/Sub credentials | No | https://www.googleapis.com/auth/pubsub
 
 ==== Publisher/Subscriber Configuration
 
-This section describes configuration options to customize the behavior of the application's Pub/Sub publishers and subscribers.
+This section describes configuration options to customize the behavior of the application's Pub/Sub publishers and subscribers. Subscriber settings can be either global or subscription-specific.
```

Comment:
We used to follow the one-sentence-per-line convention. I don't know if we still want to continue with it after Spring conventions are no longer mandatory.

Reply:
One-sentence-per-line has grown on me over time: it does make it easier to review changes, since it does not lead to line reformatting whenever a sentence length changes (also, we don't have the internal tools automatically reformatting to 80 chars in open source world).

Reply:
Done, applied the one-sentence-per-line convention.